<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgotBankPin</class>
 <widget class="QWidget" name="ForgotBankPin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Forgot Bank PIN</string>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Forgot Bank PIN</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="securityAnswerBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="SecurityQuestionBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>233</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContents</enum>
   </property>
   <property name="placeholderText">
    <string>- Please select an option -</string>
   </property>
   <item>
    <property name="text">
     <string>What is your mother's maiden name?</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>What is the name of your favorite pet?</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>In what city were you born?</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>What high school did you attend?</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>What is the name of your first school?</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>What was the make of your first car?</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>What was your favorite food as a child?</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Where did you meet your spouse?</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="selectSecQuestionLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot; style=&quot;margin:0; padding:0;&quot;&gt;Select Your&lt;/p&gt;&lt;p align=&quot;center&quot; style=&quot;margin:0; padding:0;&quot;&gt;Security Question:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="SecurityAnswerLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Answer:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="newBankPinBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>4</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="newBankPinLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>New Bank PIN:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="confirmNewBPinLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm new
Bank PIN:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmNewBPinBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>4</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>350</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
